src/Widgets/Extensions/SPARQL/Model/SparqlEndpointService.js

```diff
@@ -0,0 +1,54 @@
+import { RequestService } from '../../../../Components/Request/RequestService';
+
+/**
+ * The SPARQL Endpoint Service which contains connection information for
+ * a specific SPARQL Endpoint.
+ */
+export class SparqlEndpointService extends RequestService {
+  /**
+   * Creates a SPARQL Endpoint Service
```

Comment:
This class does not create a SPARQL endpoint but it encapsulates the information designating a SPARQL Endpoint (for later consumption i.e. submitting requests/searches).
Maybe (?) a clearer comment would here be of the form: 
extends the given configuration with all the information required for accessing the designated SPARQL endpoint.